lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -420,12 +422,14 @@ class SPIRVMemoryAccess {
   }
   SPIRVWord getMemoryAccessMask() const { return TheMemoryAccessMask; }
   SPIRVWord getAlignment() const { return Alignment; }
-  SPIRVWord getAliasing() const { return AliasInstID; }
+  SPIRVWord getAliasScopeInstID() const { return AliasScopeInstID; }
+  SPIRVWord getNoAliasInstID() const { return NoAliasInstID; }
 
 protected:
   SPIRVWord TheMemoryAccessMask;
   SPIRVWord Alignment;
-  SPIRVId AliasInstID;
+  SPIRVId AliasScopeInstID;
```

Comment:
I'd prefer not to address clang-tidy errors in these patch as it goes against existing code style in this file.